tests/image/image_test.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import (division, absolute_import, unicode_literals,
+                        print_function)
+
+import unittest
+
+from file_metadata.image.image import ImageFile
+from tests.utilities import fetch_file, mock
+
+
+class ImageFileTest(unittest.TestCase):
+
+    def setUp(self):
+        self.mona_lisa = ImageFile(fetch_file('mona_lisa.jpg'))
+        self.circles = ImageFile(fetch_file('circles.jpg'))
+        self.lines = ImageFile(fetch_file('lines.jpg'))
+
+    def test_pil(self):
+        data = self.mona_lisa.analyze_pil()
+        self.assertEqual(data['PIL:ImageWidth'], 722)
+        self.assertEqual(data['PIL:ImageHeight'], 1074)
+
+    def test_geometry(self):
+        data = self.circles.analyze_geometry()
+        self.assertEqual(round(data['Geometry:EdgeRatio'], 2), 0.06)
+        self.assertNotIn('Geometry:Lines', data)
+        circles = data['Geometry:Circles']
+        self.assertEqual(tuple(map(round, circles[0])), (147.0, 149.0, 45.0))
+        self.assertEqual(len(circles), 2)
+
+        data = self.lines.analyze_geometry()
+        self.assertEqual(round(data['Geometry:EdgeRatio'], 2), 0.03)
+        self.assertNotIn('Geometry:Circles', data)
+        self.assertEqual(len(data['Geometry:Lines']), 6)
+
```

Comment:
The code does not comply to PEP8.

*PEP8Bear, severity NORMAL, section `autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tests/image/image_test.py
+++ b/tests/image/image_test.py
@@ -33,4 +33,3 @@
         self.assertEqual(round(data['Geometry:EdgeRatio'], 2), 0.03)
         self.assertNotIn('Geometry:Circles', data)
         self.assertEqual(len(data['Geometry:Lines']), 6)
-
```